googleapis/src/main/java/io/grpc/googleapis/GoogleCloudToProdNameResolver.java

```diff
@@ -142,13 +147,20 @@ private void resolve() {
     if (resolving || shutdown || delegate == null) {
       return;
     }
+
     resolving = true;
+    if (logger.isLoggable(Level.FINE)) {
+      logger.fine("resolve with schemaOverride = " + schemeOverride);
+    }
+
     if (schemeOverride.equals("dns")) {
       delegate.start(listener);
       succeeded = true;
       resolving = false;
       return;
     }
+
+    // must be xds
```

Comment:
Did you mean to leave this here?

Reply:
Yes, I found it non-obvious when first looking at this code that not being dns meant that it was xds.